src/chrome/content/rules/Lloydsbank.com.xml

```diff
@@ -62,42 +17,15 @@ Fetch error: http://secure.lloydsbank.com/ => https://secure.lloydsbank.com/: To
 	<target host="islandscommercial.lloydsbank.com" />
 	<target host="mycarfinance.lloydsbank.com" />
 	<target host="www.mycarfinance.lloydsbank.com" />
-	<target host="lloydslink.online.lloydsbank.com" />
 	<target host="request.lloydsbank.com" />
 	<target host="resources.lloydsbank.com" />
 	<target host="secure.lloydsbank.com" />
```

Comment:
For the record, please add a test url https://secure.lloydsbank.com/online_forms/contact_us/enquiry/form.asp